components/camel-spring-rabbitmq/src/main/java/org/apache/camel/component/springrabbit/RabbitMQConstants.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.springrabbit;
+
+public final class RabbitMQConstants {
+
+    public static final String DEFAULT_EXCHANGE_NAME = "default";
+
+    public static final String CHANNEL = "CamelRabbitmqChannel";
+    public static final String ROUTING_OVERRIDE_KEY = "CamelRabbitmqRoutingOverrideKey";
```

Comment:
Does it make sense to change header keys the naming to `CamelSpringRabbitmq...` just to have it consistent with the component name?

Reply:
Yeah good idea